Ines/index.js

```diff
@@ -0,0 +1,13 @@
+const colors = ["red", 9999, "green", "blue",777, "yellow", "black", "purple", "brown"];
+
+let random = Math.floor(Math.random()*9);
+// console.log("random>",random);
+
+if (typeof colors[random] === "number"){
```

Comment:
You just delete 1 number in console either 999 or 777, I send you the solution.
const colors = [
    "red",
    9999,
    "green",
    "blue",
    777,
    "yellow",
    "black",
    "purple",
    "brown",
  ];
  const random = colors[Math.floor(Math.random() * colors.length)];
  const pickColor = isNaN(random) === true ? random : "";
  console.log(pickColor);